completers/gm_completer/cmd/time.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"github.com/carapace-sh/carapace"
+	"github.com/spf13/cobra"
+)
+
+var timeCmd = &cobra.Command{
+	Use:   "time",
+	Short: "time one of the other commands",
+	Run:   func(cmd *cobra.Command, args []string) {},
+}
+
+func init() {
+	carapace.Gen(timeCmd).Standalone()
+
+	rootCmd.AddCommand(timeCmd)
+
+	carapace.Gen(timeCmd).PositionalCompletion(carapace.ActionCommands(rootCmd))
+
+	carapace.Gen(timeCmd).PositionalAnyCompletion(carapace.ActionFiles())
+}
```

Comment:
We can fully embed the subcommands with arguments:

```suggestion
func init() {
	carapace.Gen(timeCmd).Standalone()
	timeCmd.Flags().SetInterspersed(false)

	rootCmd.AddCommand(timeCmd)

	carapace.Gen(timeCmd).PositionalAnyCompletion(
		bridge.ActionCarapaceBin("gm"),
	)
}
```